plugins/cpp_metrics/service/cxxmetrics.thrift

```diff
@@ -38,10 +38,16 @@ struct CppMetricsAstNodeSingle
   3:double value
 }
 
-struct CppMetricsAstNodeAll
```

Comment:
Was this struct unused previously, that you can change it?

Reply:
Yes, I checked, and it wasn't used anywhere.